inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors-v1.13/doris/pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.inlong</groupId>
-        <artifactId>sort-connectors</artifactId>
+        <artifactId>sort-connectors-v1.13</artifactId>
         <version>1.7.0-SNAPSHOT</version>
     </parent>
     <artifactId>sort-connector-doris</artifactId>
```

Comment:
Maybe all connector artifactId need append flink version

Reply:
There is no need to define the version, the module is closed

Reply:
> There is no need to define the version, the module is closed

@GanfengTan All connector jar will deploy to maven repository. Connector artifactId can't be distinguished when multiple flink version package and version is same.

Reply:
@gong Thank you for your suggestion. For the stability of the historical version, the 1.13 connector does not add a version identification, and other flink versions will add the corresponding version.